commit 1f1096cd2c9657fceb427db5856b5c9a53a86d3a
Author: Alois Schloegl <alois.schloegl@ist.ac.at>
Date:   Fri Feb 9 13:58:01 2018 +0100

    libbiosig v1.9.0 and higher: libbiosig2 and libbiosig are now combined in libbiosig

diff --git a/Makefile.static b/Makefile.static
index 1e45833d..b9c3b9d0 100644
--- a/Makefile.static
+++ b/Makefile.static
@@ -258,7 +258,7 @@ endif
 
 ## BIOSIG related stuff ##
 ifneq (,$(findstring WITH_BIOSIG2, $(DEFINES)))
-  LIBS  += -lbiosig2
+  LIBS  += -lbiosig
 else 
   ifneq (,$(findstring WITH_BIOSIG, $(DEFINES)))
     LIBS  += -lbiosig
diff --git a/configure.ac b/configure.ac
index a464260d..073b1f65 100644
--- a/configure.ac
+++ b/configure.ac
@@ -147,9 +147,9 @@ AM_CONDITIONAL(WITH_BIOSIGLITE, test "$with_biosiglite" = "yes")
 
 if test "$with_biosig2" = "yes" ; then
     CPPFLAGS="${CPPFLAGS} -DWITH_BIOSIG2"
-    LIBBIOSIG_LDFLAGS="-lbiosig2 -lcholmod"
+    LIBBIOSIG_LDFLAGS="-lbiosig -lcholmod"
 elif test "$with_biosig" = "yes" ; then
-    CPPFLAGS="${CPPFLAGS} -DWITH_BIOSIG"
+    CPPFLAGS="${CPPFLAGS} -DWITH_BIOSIG2"
     LIBBIOSIG_LDFLAGS="-lbiosig -lcholmod"
 elif test "$with_biosiglite" = "yes" ; then
     CPPFLAGS="${CPPFLAGS} -DWITH_BIOSIG2 -DWITH_BIOSIGLITE"
